### DISCUSSION TITLE
Make text finetuning version of Trompt, just like "transformers_text.py" for FTtransformer?

### DISCUSSION THREAD

**Discussion by hohoCode** [pyg-team/pytorch-frame#315] [Ideas]
Can you also add an example of using text finetuning (better with LORA) version of Trompt, just like "transformers_text.py" for FTtransformer?

Better to support larger models LORA finetuning such as LLMs.

This will be very useful to compare FTtransformer and Trompt after text finetuned embeddings.

Thanks!

**Comment by zechengz** [comment]
Thanks for creating this issue. We will look into this soon.

**Comment by zechengz** [comment]
Hi @hohoCode , thanks for the feedback again.
Do you want to make a contribution to enable passing `stype_encoder_dict`, such as the one in the `FTTransformer`
https://github.com/pyg-team/pytorch-frame/blob/4a6eeed1892a4756dc4fb80cd3674221c59198e8/torch_frame/nn/models/ft_transformer.py#L58-L59
to the `Trompt` (then it can use the text embeddings)?
https://github.com/pyg-team/pytorch-frame/blob/4a6eeed1892a4756dc4fb80cd3674221c59198e8/torch_frame/nn/models/trompt.py#L73-L75
Please let me know if you want to make a contribution. Thanks.

**Comment by weihua916** [comment]
@hohoCode Thanks for your suggestion! I agree it is interesting. Please feel modify the example code, experiment yourself, and let us know how `Trompt` performs!

**Reply by weihua916** [reply]
Converting this into discussion.

**Comment by hohoCode** [comment]
Thanks a lot for your response. I am totally new to the library, it will probably take me quite a while to understand the great work! So at this moment may not be able to do things here unfortunately. But I am highly interested and will keep following your exciting work along the journey!

Thanks a lot. Look forward to more great news from you all!
